### ISSUE TITLE
file architecture / organization

### ISSUE THREAD

**Issue by liangjh** [mardix/assembly#5]
Actually more of a question than an issue per se.   

Looking at the docs, the views need to be in __init__.py, and the models need to be in a __model__.py file.   Is this a hard requirement.  Let's say if I have lots of models with various business logic in them.  Do all of those models need to be in the __model__.py file or can we put them into different modules and import them all into __model__.py ?

**Comment by mardix** [comment]
Thank you very much for your interest. 

Views and Models don't have to exist in init.py or models.py.  As long as a file containing a view or a model is imported, it will be availalble. 

For views, any classes from any file that extend Assembly will work.

```
# mysuperview.py

from assembly import Assembly

class Index(Assembly):
    def index(self):
        return
```

for models, any classes from any that extend **db.Model** will work.
```
# myawesomemodels.py

from assembly import db 

class Article(db.Model):
    title = db.Column(db.String(250)
```

Remember, they still have to be imported.


Also, I just released version **1.3.0** with an updated and flatter structure to make it simpler.

**Comment by liangjh** [comment]
thanks
